formiodata/builder.py

```diff
@@ -25,6 +25,7 @@ def __init__(self, schema_json, **kwargs):
         # i18n (translations)
         self.i18n = kwargs.get('i18n', {})
         self.resources = kwargs.get('resources', {})
+        self.resources_ext = kwargs.get('resources_ext', False)
```

Comment:
I tried to get the idea (aim) of this feature.
I'm still clueless after checking the code.
Please could you describe the usage (aim)? maybe add a short description or example in the README?
Thanks!

Reply:
Hi 

[here](https://github.com/novacode-nl/python-formio-data/pull/4/commits/84c64223d3bbdf894e305d1d956d6a36f885ff04#diff-92dd155496a47c8983ad09bba2501210b1e822222469fd24381166a6f8d338f1R44) pass the method ref

[here](https://github.com/novacode-nl/python-formio-data/pull/4/commits/84c64223d3bbdf894e305d1d956d6a36f885ff04#diff-85d655ecad9c721663f280667f10d84a4669d775f771a50d9c0e9be92c580e3eR348) if a condition is valid call method to retrieve datas and make the data list

Method can execute, external request, load file, o return a list of data.